rendermodules/pointmatch/generate_point_matches_spark.py

```diff
@@ -78,11 +78,18 @@ def run(self):
         cmd = "{} --master {}".format(sparksubmit, self.args['masterUrl'])
         cmd = cmd + " --executor-memory {}".format(self.args['memory'])
         cmd = cmd + " --driver-memory {}".format(self.args['driverMemory'])
+        if self.args.get('spark_files',None) is not None:
+            for spark_file in self.args['spark_files']:
+                cmd = cmd + " --files {}".format(spark_file)
+        if self.args.get('spark_conf',None) is not None:       
+            for key,value in  self.args['spark_conf'].items():
+                cmd = cmd + " --conf {}='{}'".format(key,value)
```

Comment:
The spark submit documentation suggests wrapping the entire `<key>=<value>` in double quotes in case of spaces, but a more readable alternative in python might be to use the subprocess module.

e.g. 
```subprocess.call(['bash', 'some file.sh'])```
will work while
```os.system('bash some file.sh')``` will fail looking for a file named "some"